src/main/java/tools/jackson/databind/cfg/MapperBuilder.java

```diff
@@ -798,6 +798,27 @@ public B configure(DatatypeFeature feature, boolean state) {
         return _this();
     }
 
+    /**
+     * The builder returned uses default settings more closely
+     * matching the default configs used in Jackson 2.x versions.
+     * <p>
+     *     This method is still a work in progress and may not yet fully replicate the
+     *     default settings of Jackson 2.x.
+     * </p>
+     */
+    public B configureForJackson2() {
+        return disable(StreamReadFeature.USE_FAST_DOUBLE_PARSER)
```

Comment:
I think we should not duplicate settings for underlying `TokenStreamFactory` here, violates DRY principle, requires changes in two places (granted, once we have things set there won't be too much to sync).

But should then mention this in Javadoc, that it only changes databind level settings, not underlying Streaming.

Reply:
This is why I didn't want these methods and wanted higher level method. In a higher level method like JsonMapper.buiderWithJackson2Defaults(), I can add my own JsonFactory (with Jackson 2 defaults).

In this method, I have to contend with users using their own JsonFactory or the default (non-jackson2-like factory).

Reply:
I don't understand: if they want to provide different `JsonFactory`, let them?

But `JsonMapper.buiderWithJackson2Defaults()` can just get `JsonFactory` with Jackson 2 defaults.

Reply:
changed the configureForJackson2 methods